CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Fixed
 * Form Group: Rich content overflow ([#2600](https://github.com/dso-toolkit/dso-toolkit/issues/2600))
 * Alert: Achtergrondkleur wordt op verkeerde niveau gezet ([#2614](https://github.com/dso-toolkit/dso-toolkit/issues/2614))
+* Document List: Add aria-expanded to dropdown menu example button ([#2561](https://github.com/dso-toolkit/dso-toolkit/issues/2561))
```

Comment:
Ja, akkoord dat het inderdaad een fix voor Document List is.. Hoewel het erg grijs gebied is

Reply:
Eens, maar toch ;)